### DISCUSSION TITLE
Chatwoot Native apps for windows / Mac / Linux

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#1839] [General]
Chatwoot doesn't have native apps for Desktop usage at the moment. 
One could rely on services like https://www.todesktop.com/ if that's a requirement.

**Comment by pranavrajs** [comment]
Or try this https://webcatalog.app/catalog/chatwoot/

**Reply by iammattmartin** [reply]
This works although notifications need better support. :)

**Comment by morozover** [comment]
I need this.
In our company agents use native app for communications with customers

**Comment by nourkagha** [comment]
I just stumbled across this and after some quick research, I was able to build an Electron app for Chatwoot using [nativefier/nativefier](https://github.com/nativefier/nativefier).

Works like a charm on any platform (Windows, macOS and Linux) and looks completely native.

The only downside is it doesn't seem that Web Push is supported on Electron, so push notifications don't work. Something like [MatthieuLemoine/electron-push-receiver](https://github.com/MatthieuLemoine/electron-push-receiver) might solve this, but it's experimental.

**Reply by sojan-official** [reply]
let's track it here: https://github.com/chatwoot/chatwoot/issues/3316
